AWS/ubuntu-behind-bastion/README.md

```diff
@@ -51,14 +51,6 @@ This is an appropriate AWS key id and secret from your AWS account which Terrafo
 
 This is a ScaleFT Enrollment Token which you've created in the ScaleFT dashboard. Be sure to copy the entire enrollment token into this field!
 
-### ~/.ssh/windeployer-rsa.pub
```

Comment:
Aha, not sure if this was deleted intentionally or a consequence of how this was committed, but seems ok